coil-compose-core/src/commonMain/kotlin/coil3/compose/CrossfadePainter.kt

```diff
@@ -45,6 +45,7 @@ class CrossfadePainter(
     val timeSource: TimeSource = TimeSource.Monotonic,
     val fadeStart: Boolean = true,
     val preferExactIntrinsicSize: Boolean = false,
+    val preferEndFirstIntrinsicSize: Boolean = false
```

Comment:
We'll need to add a `@Deprecated("Kept for binary compatibility.", level = DeprecationLevel.HIDDEN)` version of this constructor without the `preferEndFirstIntrinsicSize` argument to preserve binary compatibility. Check out `NetworkClient` for an example .

Reply:
50ab6102543d524ed9f7e15a04b36c2bf7c7c9bc
please check this